codes/classical/q-ary_digits/frameproof.yml

```diff
@@ -0,0 +1,120 @@
+  #encoders:
+    #- 'Specific description of a process that makes the states, usually for quantum codes.'
+    #- 'Unitary circuit of depth ... \cite{arxiv:old-paper}.'
+    #- 'Measurement-based preparation ... with ancilla overhead of ... .'
+    #- 'Leave discussion of fault tolerance to fault-tolerance field.'
+
+  # Not all fields are indexed by a dash
+  #transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  #general_gates:
+    #- 'Universal gate set achieved by either additional ... gate.'
+    #- 'Magic-state distillation protocols'
+    #- 'kth Clifford hierarchy gates obtained by ... circuits'
+
+  #decoders:
+    #- 'Details about how syndrome measurements are done; discuss overhead, if applicable.'
+    #- 'MWPM decoding algorithm \cite{doi:good-paper} with ... overhead.'
+    #- 'Just-in-time decoder with ... \cite{arxiv:awesome-paper}.'
+
+  #fault_tolerance:
+    #- 'Transversal gates are fault-tolerant w.r.t. ... noise \cite{doi:ok-paper}'
+    #- 'Other fault-tolerant gadgets (measurements, encoders, error correcting steps)'
+    #- 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+    #- 'Pieceable fault tolerance.'
+
+  #code_capacity_threshold:
+    #- '\(1.5%\) error-correction threshold against some noise with *noiseless* decoder of some complexity \cite{arxiv:paper}.'
+    #- '\(5.2\%\) asymptotic upper bound from some estimation algortihm (Monte Carlo, e.g.) that may or may not have a feasible decoder.'
+
+  #threshold:
+    #- '\(0.3\%\) error-correction threshold ... with *noisy* ... decoder of some complexity \cite{doi:good-paper}.'
+    #- '\(10^{-5}\) computational threshold using concatenated scheme under ... noise with overhead of ... '
+    #- '\(4.5\%\) asymptotic lower bound on fault-tolerant threshold with ... noise, ... overhead, and ... complexity.'
+    #- '\(6.7\%\) threshold using magic-state distillation protocols assuming ... Clifford gates.'
+
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - 'Frameproof codes are utilized in digital fingerprinting and watermarking \cite{doi:10.1137/S0895480101384633}.'
+
+notes:
+  - 'Any c-frameproof code must have at least length c \cite{doi:10.1007/3-540-44750-4_36}.'
+  - 'A length l q-ary c-frameproof code has at most \(tq^{\lceil l/c \rceil} + O(q^{\lceil l/c \rceil - 1})\) codewords where t is an integer between 1 and c and t is 1 mod c \cite{doi:10.1137/S0895480101384633}.'
+
+#relations:
+  #parents: #will add relations when IPP and traceability codes have been created and accepted into the zoo
```

Comment:
Cousins: Codes with the identifiable parent property, secure frameproof codes, separating codes, traceability codes.